tests/integration/fuzzing/parameterized_test.cairo

```diff
@@ -82,3 +82,21 @@ func test_check_exact_example{syscall_ptr: felt*, range_check_ptr}(a: felt, b: f
     return ();
 }
 
+
+@external
+func setup_negative_example() {
+    %{
+        example(a=11)
+    %}
+    return ();
+}
+
+@external
+func test_negative_example{syscall_ptr: felt*, range_check_ptr}(a: felt) {
+    %{
+        assert ids.a == 12
+    %}
```

Comment:
```suggestion
    assert a = 12;
```

Reply:
Actually that should do the job. Good idea.